GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
```

Comment:
Role is more about what a group or user would be performing. This is useful especially when you have a role that spans multiple groups or multiple roles within a single group. It allows for flexibility depending on how an organization is set up. For example the `management` role can spread across multiple groups; while within the operations group you can have `admin` and `support` roles. A group or a user can have a role. Then you can have permissions that are based on the role.